workspaces/Cargo.toml

```diff
@@ -39,11 +39,11 @@ near-crypto = "0.20.0"
 near-primitives = "0.20.0"
 near-jsonrpc-primitives = "0.20.0"
 near-jsonrpc-client = { version = "0.8", features = ["sandbox"] }
-near-sandbox-utils = "0.7.0"
+near-sandbox-utils = {git = "https://github.com/near/sandbox", tag = "v0.7.1"}
```

Comment:
Just for the context, due to the missing near-sandbox 1.39.0 pre-built binaries, I had a little detour: https://near.zulipchat.com/#narrow/stream/295306-contract-runtime/topic/neard.20sandbox.20for.20performance.20analysis

I will release near-sandbox 0.8.0 with nearcore 1.38.0 today

Reply:
updated